crates/node/src/commands/run.rs

```diff
@@ -29,6 +29,14 @@ pub struct ExtendedRunCmd {
     /// Choose a supported DA Layer
     #[clap(long)]
     pub da_layer: Option<DaLayer>,
+
+    /// When enabled, more information about the blocks and their transaction is cached and stored
+    /// in the database.
+    ///
+    /// This may improve response times for RPCs that require that information, but it also
+    /// increases the memory footprint of the node.
+    #[clap(long, action)]
```

Comment:
what does `action` do ?

Reply:
We want it to be false if not passed through CLI, is it what it does?

Reply:
`action` on `bool` makes it so that when the flag is automatically `false` by default and then replaced by `true` when the flag is set. I just checked the doc and apparently it's automatically on for `bool` now. I don't think that was the case before.

Do I remove this?